### ISSUE TITLE
0.2.0 doesn't compile

### ISSUE THREAD

**Issue by achesnais** [metosin/muuntaja#39]
# Problem

I found this while initiating a new luminus app. The app won't run or compile. Steps to reproduce and error message below:

```{bash}
lein new luminus guestbook +h2
cd guestbook
lein run migrate

Exception in thread "main" java.lang.ExceptionInInitializerError
	at clojure.main.<clinit>(main.java:20)
Caused by: java.io.FileNotFoundException: Could not locate msgpack/clojure_extensions__init.class or msgpack/clojure_extensions.clj on classpath. Please check that namespaces with dashes use underscores in the Clojure file name., compiling:(muuntaja/format/json.clj:1:1)
...
```

This is caused by [the breaking change](https://github.com/metosin/muuntaja/blob/master/CHANGELOG.md) in the latest version.

# Investigations

I went ahead and cloned muuntaja, then ran `lein with-profile uberjar check` and sure enough, I got

```{bash}
Exception in thread "main" java.io.FileNotFoundException: Could not locate msgpack/clojure_extensions__init.class or msgpack/clojure_extensions.clj on classpath
```

I'm pretty sure that's not expected behaviour? I would have submitted a fix but since you don't want the dep in the base dependencies, I'm not sure what I can do.

**Comment by ikitommi** [comment]
Oh. There were dependencies to the msgpack namespaces from wrong places. Didn't see those as the deps where present (and tested) in the dev profile. Should be fixed now, could you test `0.2.1-20170402.095947-1`. If that works, will ship it.

**Comment by ikitommi** [comment]
tested myself, works now. Shipped `0.2.1`. thanks for reporting.

**Comment by achesnais** [comment]
Thanks for tackling so quickly!
